resources/tentative/editors/assets/longtext.html

```diff
@@ -1,200 +1,3 @@
-<!DOCTYPE html>
-<html lang="fr">
-    <head>
-        <meta charset="utf-8" />

```

Comment:
Why are we removing these HTML / CSS?

Reply:
This was a comment from @bgrins in the patch that added this workload: these are not displayed at all so it's not useful to add them. But feel free to disagree!

Reply:
See https://github.com/WebKit/Speedometer/pull/186#discussion_r1204671489

But the PR was merged before I could fix it.

Reply:
Ok, thanks for the explanation.